docs/getting-started-tutorial/4-css-selector-transforms.adoc

```diff
@@ -0,0 +1,171 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# CSS Selector Transforms
+
+Because Lift operates by transforming HTML trees, it needs an easy way to
+specify those transformations. Otherwise we'd be doing a bunch of recursive
+tree searches and munges and it would get ugly and probably slow to boot. To
```

Comment:
It could just be how I'm reading it today, but the end of this sentence feels a bit awkward. Would recommend changing `ugly and probably slow to boot` to `ugly, unpleasant, and time consuming`.

Reply:
Yeah I went too conversational here, trying to go for a rhythm that shows all the bad things snowballing haha.